cmd/ndm_daemonset/probe/addhandler.go

```diff
@@ -124,6 +124,7 @@ func (pe *ProbeEvent) addBlockDevice(bd blockdevice.BlockDevice, bdAPIList *apis
 			return nil
 		}
 	} else {
+		// FIXME: update controller cache after setting uuid
```

Comment:
Is this meant to be addressed in this PR? or kept for a later one.

Reply:
Also, why is the update in the cache required here?

Reply:
Oh this is because the block device stored in the cached does not have the uuid set even though we have generated one in this part of the code. If you notice [here](https://github.com/openebs/node-disk-manager/pull/595/files#diff-a58c9edd278ce8e7038021bcb3215de629d9262064dd13c69933bfa5aebc5cd6R29) in `changeBlockDevice`, I am generating the uuid if its not set. This is happening because the bd in cache doesn't have the uuid field set since while adding we are generating the uuid after updating the cache.

Reply:
I assume this is a separate issue, but I can fix it in this PR if it's fine to do so.

Reply:
@akhilerm wdyt?

Reply:
yep. separate PR is good.